src-docs/src/views/context_menu/context_menu_example.js

```diff
@@ -76,6 +80,34 @@ export const ContextMenuExample = {
       ),
       demo: <SinglePanel />,
     },
+    {
+      title: 'Sizes',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: smallSizeSource,
+        },
+        {
+          type: GuideSectionTypes.HTML,
+          code: smallSizeHtml,
+        },
+      ],
+      text: (
+        <div>
+          <p>
+            You can reduce the size of <strong>EuiContextMenu</strong> by
+            setting <EuiCode>size</EuiCode> to <EuiCode>s</EuiCode>.
+          </p>
+          <p>
+            Use the default size <EuiCode>m</EuiCode> for major actions such as
+            Share and Export which appear in top application menu items. Use the
+            smaller size <EuiCode>s</EuiCode> for actions that affect only a
+            single item and are repeated per row.
+          </p>
```

Comment:
Just cleaning up some verbage here and combining to make it a little more succinct.
```suggestion
          <p>
            <strong>EuiContextMenu</strong> supports a small and medium <EuiCode>size</EuiCode>.
            Use the default size <EuiCode>m</EuiCode> for major actions such as
            <em>Share</em> and <em>Export</em> which appear in top application menus. Use the
            smaller size <EuiCode>s</EuiCode> for actions that affect only a
            single item or are repeated like in EuiTable actions.
          </p>
```